docs/backend/guides/dtod/testing-dtod.md

```diff
@@ -0,0 +1,51 @@
+# Testing DTOD
+
+The instructions that follow will say to change the environment variable inside of `.envrc`, however, there are other methods one can take to update to environment variables.
+
+- Environment variables can be changed inside of `.envrc.local`
+- Runing `export DTOD_USE_MOCK=false` after `direnv allow`
+
+:::note
+If you run the server in Goland, be sure to refresh the Goland environment.
```

Comment:
**note**: A plugin I've been testing out is https://plugins.jetbrains.com/plugin/19275-better-direnv and so far it seems to work for the last few times I changed the `.envrc*` files.